src/routes/reviews.ts

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express';
+import * as reviewController from '../controllers/reviewController';
+import auth from '../middleware/auth';
+
+const router = Router();
+
+router.get('/', reviewController.indexReviewFuncs);
```

Comment:
High chance this endpoint will be unused. When would I want to get all kinds of reviews back that are binded to different users or stalls?

Reply:
I think a better design would be to bind the routes under stalls, 
e.g. `GET /stalls/:id/reviews` -> Get all the reviews for a single stall
`POST /stalls/:id/reviews` -> Create review for a stall, the `stallId` can be read through here without frontend inputting `stallId` in the request. -> High chance they are already on the stall page anyway.
`GET /reviews/:id`, `PUT /reviews/:id` and `DELETE /reviews/:id` should stay the same.

No need to change the routes for now, can change once frontend requests for it.

Reply:
If that is the case, where should i put the route definition for the GET /stalls/:id/reviews and POST/stalls/:id/reviews? Under the stall rotues or reviews routes?